packages/charm/src/manager.ts

```diff
@@ -340,7 +340,10 @@ export class CharmManager {
     if (isCell(id)) charm = id;
     else charm = this.runtime.getCellFromEntityId(this.space, { "/": id });
 
-    await this.runtime.storage.syncCell(charm);
+    await this.runtime.storage.syncCell(charm, false, {
```

Comment:
we could write this as `charm.asSchema(charmSchema)` (or even narrowSchema once we support that)